tests/Tests.AzureAppConfiguration/Tests.AzureAppConfiguration.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.0" />
```

Comment:
>2.0.0 [](start = 81, length = 5)

We could use the updated patched version `2.0.2`

Reply:
It looks like the latest one is 3.1.5. Should we just update it to the latest version?
[https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Json](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Json) 

---
In reply to: [453142063](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r453142063) [](ancestors = 453142063)

Reply:
We generally avoid using any package version that has updated patched versions available. If this was product code, we would prefer using the older version with the latest patch to include any fixes or security updates. For test code, you can choose `2.0.2` or most-recent patch for any newer version.

---
In reply to: [453945711](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r453945711) [](ancestors = 453945711,453142063)

Reply:
Changed it to 2.0.2

---
In reply to: [454039139](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r454039139) [](ancestors = 454039139,453945711,453142063)